.github/workflows/e2e.yaml

```diff
@@ -94,42 +80,37 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
+      - name: Setup LXD
+        uses: canonical/setup-lxd@v0.1.3
+        with:
+          bridges: "lxdbr0"
+      - name: Configure LXD
+        run: |
+          sudo ./hack/setup-lxd.sh
       - name: Install requirements
         run: |
           sudo apt update
           sudo apt install make
-          sudo apt install docker-buildx
           sudo snap install kubectl --classic --channel=1.32/stable
       - name: Download artifacts
         uses: actions/download-artifact@v4
         with:
           name: e2e-images
           path: .
-      - name: Load provider image
-        run: sudo docker load -i provider-images.tar
-      - name: Load k8s-snap old image
-        run: |
-          sudo docker load -i k8s-snap-image-old.tar
-      - name: Load k8s-snap new image
-        if: matrix.ginkgo_focus == 'Workload cluster upgrade'
-        run: |
-          sudo docker load -i k8s-snap-image-new.tar
-      - name: Create docker network
-        run: |
-          sudo docker network create kind --driver=bridge -o com.docker.network.bridge.enable_ip_masquerade=true
-      - name: Increase inotify watches
+      - name: Setup bootstrap cluster
         run: |
-          # Prevents https://cluster-api.sigs.k8s.io/user/troubleshooting#cluster-api-with-docker----too-many-open-files
-          sudo sysctl fs.inotify.max_user_watches=1048576
-          sudo sysctl fs.inotify.max_user_instances=8192
+          sudo ./hack/setup-bootstrap-cluster.sh bootstrap-cluster 1.32 ./provider-images.tar
```

Comment:
I assume the `provider-images.tar` is passed so it can be side-loaded into management cluster right?